main/libzdb/APKBUILD

```diff
@@ -27,13 +27,13 @@ build() {
 
 check() {
 	cd "$builddir"
-	make verify
+	yes . | make verify
```

Comment:
Adding a comment about what `yes .` is doing here would be helpful.

Reply:
Added a lengthy explanation